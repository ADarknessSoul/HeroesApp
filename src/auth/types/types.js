//El proposito de este archivo es mantener centralizado los tipos de acciones que se podr√°n ejecutar en el reducer

export const types = {

    login: '[Auth] Login',
    logout: '[Auth] Logout',
    Admin: '[Auth] Admin',

}