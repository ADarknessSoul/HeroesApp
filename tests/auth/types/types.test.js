import { screen } from "@testing-library/react";
import { types } from "../../../src/auth/types/types";

describe('Pruebas en types.js', ()=> {

    test('Debe regresar los types iniciales', ()=> {

        expect(types).toMatchSnapshot();

    }); 

});