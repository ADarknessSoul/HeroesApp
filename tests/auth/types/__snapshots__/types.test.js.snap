// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en types.js Debe regresar los types iniciales 1`] = `
{
  "login": "[Auth] Login",
  "logout": "[Auth] Logout",
}
`;
